torch/distributed/rpc/backend_registry.py

```diff
@@ -1,24 +1,37 @@
 from __future__ import absolute_import, division, print_function, unicode_literals
 
 import collections
+import datetime
 import enum
 
 import torch.distributed as dist
 import torch.distributed.distributed_c10d as dc10d
 
+from . import constants as rpc_constants
 
-BackendValue = collections.namedtuple("BackendValue", ["init_backend_handler"])
+
+BackendValue = collections.namedtuple(
+    "BackendValue", ["construct_rpc_agent_options_handler", "init_backend_handler"]
+)
 
 # Create an enum type, `BackendType`, with empty members.
 BackendType = enum.Enum(value="BackendType", names={})
 
 
-def register_backend(backend_name, init_backend_handler):
+def register_backend(
+    backend_name, construct_rpc_agent_options_handler, init_backend_handler
+):
     """Registers a new RPC backend.
 
     Arguments:
         backend (str): backend string to identify the handler.
```

Comment:
`backend_name` or rename the argument as `backend` to make them consistent

Reply:
Updating the comment to `backend_name`, calling it "name" because it's a string, not a backend enum type member.